### DISCUSSION TITLE
Erros de compilação

### DISCUSSION THREAD

**Discussion by crdavid179** [balta-io/2801#13] [Perguntas e Respostas] [answered]
Olá, sou iniciante na área e encontrei alguns erros ao digitar o comando "dotnet build"

FALHA da compilação.

C:\Users\crdav\MeuApp\obj\Debug\net5.0\.NETCoreApp,Version=v5.0.AssemblyAttributes.cs(4,12): error CS0579: Duplicar atributo "global::System.Runtime.Versioning.TargetFrameworkAttribute" [C:\Users\crdav\MeuApp\MeuApp.csproj]
C:\Users\crdav\MeuApp\obj\Debug\net5.0\MeuApp.AssemblyInfo.cs(13,12): error CS0579: Duplicar atributo "System.Reflection.AssemblyCompanyAttribute" [C:\Users\crdav\MeuApp\MeuApp.csproj]
C:\Users\crdav\MeuApp\obj\Debug\net5.0\MeuApp.AssemblyInfo.cs(14,12): error CS0579: Duplicar atributo "System.Reflection.AssemblyConfigurationAttribute" [C:\Users\crdav\MeuApp\MeuApp.csproj]
C:\Users\crdav\MeuApp\obj\Debug\net5.0\MeuApp.AssemblyInfo.cs(15,12): error CS0579: Duplicar atributo "System.Reflection.AssemblyFileVersionAttribute" [C:\Users\crdav\MeuApp\MeuApp.csproj]
C:\Users\crdav\MeuApp\obj\Debug\net5.0\MeuApp.AssemblyInfo.cs(16,12): error CS0579: Duplicar atributo "System.Reflection.AssemblyInformationalVersionAttribute" [C:\Users\crdav\MeuApp\MeuApp.csproj]
C:\Users\crdav\MeuApp\obj\Debug\net5.0\MeuApp.AssemblyInfo.cs(17,12): error CS0579: Duplicar atributo "System.Reflection.AssemblyProductAttribute" [C:\Users\crdav\MeuApp\MeuApp.csproj]
C:\Users\crdav\MeuApp\obj\Debug\net5.0\MeuApp.AssemblyInfo.cs(18,12): error CS0579: Duplicar atributo "System.Reflection.AssemblyTitleAttribute" [C:\Users\crdav\MeuApp\MeuApp.csproj]
C:\Users\crdav\MeuApp\obj\Debug\net5.0\MeuApp.AssemblyInfo.cs(19,12): error CS0579: Duplicar atributo "System.Reflection.AssemblyVersionAttribute" [C:\Users\crdav\MeuApp\MeuApp.csproj]
C:\Users\crdav\MeuApp\Program.cs(5,11): error CS0101: O namespace "MeuApp" já contém uma definição para "Program" [C:\Users\crdav\MeuApp\MeuApp.csproj]
C:\Users\crdav\MeuApp\Program.cs(7,21): error CS0111: Tipo "Program" já define um membro chamado "Main" com os mesmos tipos de parâmetro [C:\Users\crdav\MeuApp\MeuApp.csproj]
    0 Aviso(s)
    10 Erro(s)

**Comment by crdavid179** [comment]
Se alguém puder ajudar agradeço desde já

**Comment by andrebaltieri** [comment]
Post o código do seu `Program.cs`

**Comment by crdavid179** [comment]
Como assim, não entendi...

**Comment by crdavid179** [comment]
![erro 2](https://user-images.githubusercontent.com/90929157/133834704-da5a251a-2a0d-4cfb-b0e0-3e912644569e.png)

**Reply by andrebaltieri** [reply]
Bom dia @crdavid179 preciso ver o código C# que você escreveu, para identificar o problema e poder te ajudar!

**Comment by crdavid179** [comment]
Eu coloquei dotnet build dentro da pasta MeuApp


Em sáb., 18 de set. de 2021 às 10:44, Andre Baltieri <
***@***.***> escreveu:

**Comment by crdavid179** [comment, marked as answer]
Não consigo prosseguir com o curso por conta desse erro, estou na aula de Debug de .NET...

**Reply by andrebaltieri** [reply]
Boa tarde @crdavid179 preciso ver o código C# que você escreveu, para identificar o problema e poder te ajudar!

**Comment by crdavid179** [comment]
Professor... eu já mandei, eu escrevi somente o que você pediu no curso, dotnet build para compilar, dentro da pasta MeuApp...

**Comment by crdavid179** [comment]
Irei mandar novamente o print

**Reply by andrebaltieri** [reply]
Eu preciso ver o código... o conteúdo do arquivo `Program.cs`!

**Comment by crdavid179** [comment]
![erroo](https://user-images.githubusercontent.com/90929157/134536591-39934c6f-ca3c-46e6-bc6e-36c546168cb5.png)

**Comment by crdavid179** [comment]
Bom, desculpe mas não consegui comprender, pois eu não fiz nada demais, eu apenas abri o windows terminal, coloquei cd MeuApp e depois dotnet build e apareceu esses erros.

**Comment by crdavid179** [comment]
Terei de pular essa parte

**Reply by andrebaltieri** [reply]
Boa tarde @crdavid179 preciso ver o conteúdo do arquivo `Program.cs` para poder te ajudar!
